index.js

```diff
@@ -521,6 +541,12 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this &&
+      (!this.valueOf() || other.valueOf());
+  }
+
```

Comment:
This is an interesting conundrum. Let's discuss it in https://github.com/sanctuary-js/sanctuary-type-classes/pull/46#discussion_r110620730.